target_eventbridge/helpers/constantsHelper.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+AWS_SERVICE_NAME = "operations-travel"
+Region = os.environ.get("AWS_REGION")
+AWS_REGION = f'{Region}'
+
+# Client config used by both Eventbridge.
+AWS_CLIENT_CONFIG = { "region": AWS_REGION }
+
+# Event Bus name used by the gapiMockData.request.ts file in order to route the payload to the expected location
+AWS_BUS_NAME = "operationsEventBus"
+Environment = os.environ.get("Environment")
+AWS_EVENT_BUS_NAME = f"{Environment}_{AWS_BUS_NAME}"
+
+# Payload Source
+PAYLOAD_SOURCE_EMPLOYEE = "people.employee"
+PAYLOAD_SOURCE_PAYROLL = "people.payroll-result-line"
+PAYLOAD_SOURCE_OPERTATIONS_EGENCIA = "operations.egencia"
+
+# Payload detailTypes
+PAYLOAD_DETAILTYPE_EMPLOYEE_CHANGE = "employee.change"
```

Comment:
Do we need all of these employee data types?

Reply:
I think we might want all of these constants to live in the orchestrator, not here. This target could be used for pushing to any event bridge, not just ours.